src/sage/schemes/elliptic_curves/ell_field.py

```diff
@@ -1588,6 +1588,62 @@ def isogenies_prime_degree(self, l=None, max_l=31):
         from .isogeny_small_degree import isogenies_prime_degree
         return sum([isogenies_prime_degree(self, d) for d in L], [])
 
+    def isogenies_degree(self, n):
+        """
+        Return a list of all separable isogenies of given degree with domain
+        equal to ``self``, which are defined over the base field of ``self``.
+
+        INPUT:
+
+        - ``n`` -- an integer.
+
+        TESTS::
+            sage: E = EllipticCurve(GF(11), [1, 1])
+            sage: E.isogenies_degree(23 * 19)
+            [Composite morphism of degree 437 = 1*19*23:
+               From: Elliptic Curve defined by y^2 = x^3 + x + 1 over Finite Field of size 11
+               To:   Elliptic Curve defined by y^2 = x^3 + 8*x + 7 over Finite Field of size 11,
+             Composite morphism of degree 437 = 1*19*23:
+               From: Elliptic Curve defined by y^2 = x^3 + x + 1 over Finite Field of size 11
+               To:   Elliptic Curve defined by y^2 = x^3 + 2*x + 6 over Finite Field of size 11,
+             Composite morphism of degree 437 = 1*19*23:
+               From: Elliptic Curve defined by y^2 = x^3 + x + 1 over Finite Field of size 11
+               To:   Elliptic Curve defined by y^2 = x^3 + 6*x + 2 over Finite Field of size 11,
+             Composite morphism of degree 437 = 1*19*23:
+               From: Elliptic Curve defined by y^2 = x^3 + x + 1 over Finite Field of size 11
+               To:   Elliptic Curve defined by y^2 = x^3 + 7*x + 8 over Finite Field of size 11]
+
```

Comment:
```suggestion

        ::

```
...to render unrelated examples in separate boxes.